modules/core/src/de/diedavids/cuba/dataimport/service/ImportWizardServiceBean.groovy

```diff
@@ -39,4 +41,39 @@ class ImportWizardServiceBean implements ImportWizardService {
 
         dataManager.commit(commitContext)
     }
+
+    @Override
+    Collection<ImportConfiguration> getImportConfigurations(MetaClass metaClass) {
+        if(metaClass==null)
+            return Collections.emptyList();
+
+        def entityName = metaClass.name
+
+        LoadContext loadContext =
+                LoadContext.create(ImportConfiguration.class)
+                .setQuery(
+                LoadContext.createQuery('select o from ddcdi$ImportConfiguration o where o.entityClass = :entityName')
+                .setParameter("entityName", entityName))
+                .setView("importConfiguration-view")
+
+        return dataManager.loadList(loadContext);
+    }
+
+    @Override
+    ImportConfiguration getImportConfigurations(MetaClass metaClass, String configName) {
+        if(metaClass==null)
+            return Collections.emptyList();
+
+        def entityName = metaClass.name
+
+        LoadContext loadContext =
+                LoadContext.create(ImportConfiguration.class)
+                        .setQuery(
+                        LoadContext.createQuery('select o from ddcdi$ImportConfiguration o where o.entityClass = :entityName AND o.name = :cName')
+                                .setParameter("entityName", entityName)
+                                .setParameter("cName", configName))
+                        .setView("importConfiguration-view")
+
+        return dataManager.loadList(loadContext).first();
```

Comment:
I changed it with `getImportConfigurationByName` since this is returning only one `ImportConfiguration`.
Yes, I used `.first()` to get only one. We can update this later based on the use.  
This method is ment to be used from the browser Entity